File: Prac Code/practice/app/src/main/java/com/zoey/android/practice/MainActivity.java
package com.zoey.android.practice;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    private Intent nextPage;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        nextPage=new Intent(this,FullscreenActivity.class);


    }

    public void newActivity(View view) {
        startActivity(nextPage);
    }
}

File: _AndroidTestField/WeatherTest/app/src/main/java/com/example/weatherapi_test/MainActivity.kt
package com.example.weatherapi_test

import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch
import java.net.HttpURLConnection
import java.net.URL
import kotlin.coroutines.CoroutineContext

class MainActivity : AppCompatActivity() {
    lateinit var uriTV : TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        uriTV = uriTV?.findViewById(R.id.uriTV)

        val API_KEY = "ef7108872ae71fd5fd237a0cdd318096"
        val DEFAULT_API = "api.openweathermap.org/data/2.5/weather"

        var uri :Uri = Uri.parse("api.openweathermap.org/data/2.5/forecast?id=524901").buildUpon()
                .appendQueryParameter("APPID",API_KEY).build()



    }


    fun weatherAPI_Call (){

    }
}
